ui/raidboss/data/02-arr/raid/t10.txt

```diff
@@ -4,70 +4,70 @@
 hideall "--Reset--"
```

Comment:
If `Reset` is renamed to `reset`, should this `hideall` also use `reset`?

Reply:
No, I think I might have not re-run the script to upper-case all the `Resets`. I'll just do that...